bitnami/cert-manager/values.yaml

```diff
@@ -201,6 +201,18 @@ controller:
   ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
   ##
   podAnnotations: {}
+  ## @param controller.dnsPolicy Pod DNS policy
+  ## Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  ##
+  dnsPolicy: ClusterFirst
```

Comment:
`ClusterFirst` is the default value in k8s, so rather than hardcoding it in the values, I would set it to `""`.

Reply:
Removed, and updated README file.